invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/DocumentDetails/components/DocumentItems/DocumentItems.js

```diff
@@ -32,29 +23,35 @@ export default class DocumentItems extends Component {
     return <SeeAllButton clickHandler={goToHandler(path)} />;
   };
 
-  prepareData(data) {
-    return data.hits.map(row => {
-      const entry = formatter.item.toTable(row);
-      return pick(entry, [
-        'ID',
-        'Barcode',
-        'Status',
-        'Medium',
-        'Location',
-        'Shelf',
-      ]);
-    });
-  }
+  viewDetails = ({ row }) => {
+    return (
+      <Button
+        as={Link}
+        to={BackOfficeRoutes.itemDetailsFor(row.metadata.pid)}
+        compact
+        icon="info"
+        data-test={row.metadata.pid}
+      />
+    );
+  };
 
   renderTable(data) {
-    const rows = this.prepareData(data);
-    rows.totalHits = data.total;
+    const columns = [
+      { title: '', field: '', formatter: this.viewDetails },
+      { title: 'ID', field: 'metadata.pid' },
+      { title: 'Barcode', field: 'metadata.barcode' },
+      { title: 'Status', field: 'metadata.status' },
+      { title: 'Medium', field: 'metadata.medium' },
+      { title: 'Location', field: 'metadata.internal_location.name' },
+      { title: 'Shelf', field: 'metadata.shelf' },
```

Comment:
Maybe it would be nice to show directly the current circulation status? If on loan, display e.e. ON LOAN, clickable, and I can navigate to the loan.
What do you think?